src/AudioAnalysisTools/Events/EventCommon.cs

```diff
@@ -47,6 +50,11 @@ public abstract class EventCommon : EventBase, IDrawableEvent
         /// </summary>
         public Interval<double> ScoreRange { get; set; }
 
+        /// <summary>
+        /// Gets a score in the range 0,1.
+        /// Up to user to determine a suitable range maximum.
+        /// Minimum of range assumed to be zero.
+        /// </summary>
         public double ScoreNormalised => this.Score / this.ScoreRange.Maximum;
```

Comment:
The calculation of `ScoreNormalized` is only partially complete. Use the lower bound of the provided interval in the calculation and remove the assumption of zero.